graylog2-server/src/main/java/org/graylog/events/processor/EventDefinitionDto.java

```diff
@@ -128,6 +129,10 @@ public abstract class EventDefinitionDto extends ScopedEntity implements EventDe
     @Nullable
     public abstract EventDefinitionContextService.SchedulerCtx schedulerCtx();
 
+    @JsonProperty(FIELD_ENABLED)
+    @Nullable
+    public abstract Boolean enabled();
```

Comment:
```suggestion
    @JsonProperty(FIELD_ENABLED)
    public abstract boolean enabled();
```

Small point and definitely not required, but we might be able to make this field to be non-nullable by specifying a `false` default value for it in the builder method. That might make the field slightly easier to use without needing the null checks.

If that causes any issues, we could just keep it as-is with the `@Nullable`.

```
        public static Builder create() {
            return new AutoValue_EventDefinitionDto.Builder()
                    .fieldSpec(ImmutableMap.of())
                    .notifications(ImmutableList.of())
                    .storage(ImmutableList.of())
                    .enabled(false);
        }
```

Reply:
Yes, please use a default instead of making the field nullable. Thanks!

Reply:
Updated in latest commits